src/export/CodeSystemExporter.ts

```diff
@@ -74,9 +79,9 @@ export class CodeSystemExporter {
       try {
         rule.path = this.findConceptPath(codeSystem, rule.pathArray);
         successfulRules.push(rule);
-        // We can only know that a CaretRule is meant to represent codes
-        // after it has been applied on a CodeSystem, so set the flag here
-        rule.isCodeCaretRule = true;
+        if (rule.path && rule.path !== '.') {
```

Comment:
The special case for `.` on `rule.path` is for the top-level `CodeSystem` element on the corresponding SD. I'm not sure if this is a use case that ever comes into play for CodeSystems defined in FSH, but [it does seem to exist](https://build.fhir.org/codesystem-definitions.html#CodeSystem). Do you think I should take this out?

Reply:
I think if we were _profiling_ `CodeSystem`, then `.` makes sense because you're accessing the underlying `StructureDefinition`.   But in the case here, we're actually trying to access properties of the underlying `CodeSystem` resource itself, not the SD.  In the context of an underlying `CodeSystem`, I'm not sure what `.` would represent.  So I think I agree w/ @mint-thompson on this one.